### PR TITLE
Translations201804

### PR DESCRIPTION
**Pull request by UdjinM6**
Also updated `dash_en.ts` on Transifex, so translators can start doing their part ;)

- [x] bg
- [x] de
- [x] es
- [x] fi
- [x] fr
- [x] it
- [ ] ja
- [x] nl
- [x] pl
- [ ] pt
- [ ] pt_BR
- [X] ru
- [x] sk
- [x] vi
- [x] zh_CN
- [x] zh_TW

### CODE REVIEW

**Comment by akijuh** [comment]
@UdjinM6 
Finnish updated, i fixed a lot of typos.
Please reupdate/repull FI.

File: src/qt/locale/dash_ru.ts [hunk 1]
```diff
@@ -2061,6 +2077,59 @@ https://www.transifex.com/projects/p/dash/</translation>
...
+        <source>&amp;Save Image...</source>
+        <translation>&amp;Сохранить изображение...</translation>
+    </message>
+    <message>
+        <source>&amp;Copy Image</source>
+        <translation>&amp;Копировать изображение</translation>
+    </message>
+    <message>
+        <source>Save QR Code</source>
+        <translation>Сохранить QR-код</translation>
```
Comment (gladcow) [inline src/qt/locale/dash_ru.ts]: It is named `QR код` (without `-`) in other places (line 2111,  for example). May be it is better to unify it?

**Comment by UdjinM6** [comment]
Added it, pl, zh_TW

**Comment by UdjinM6** [comment]
Added es, pt_BR (97.5%) and updated outdated list in the Makefile.am

**Comment by UdjinM6** [comment]
Since there is no significant progress on transifex, I'm dropping [WIP] prefix and would like to merge this one as is. Going to open new PR for `ja`, `pt`, `pt_BR` (and other if there are going to be some updates) later.

Pls review/ACK.

**Pull request merged**